en/technical-reference/going-live.md

```diff
@@ -3,30 +3,37 @@
 
 This document is designed to assist you in switching from *Sandbox* environment to *Production* environment, in order to start accepting real world payment for your business. This is also known as "Going Live".
 
+?>Make sure you have already completed the test payment using our [Payment Simulator](/en/technical-reference/sandbox-test.md) in the Sandbox environment to ensure that your implementation is working fine.
 
-## 1. Login to Your Midtrans Production Environment Dashboard
+## 1. Checking Your Production Account
+To make sure your payment channels on Production account is already active, follow the steps given below.
 
-To login to your Midtrans *Production* environment, follow the steps given below.
+1. Login to your Midtrans Administrative Portal using *Production* environment at https://dashboard.midtrans.com/login.
 
-1. Login to your account at https://account.midtrans.com. 
-    - refer to [Access Midtrans Administration Portal](/en/midtrans-account/overview.md#accessing-midtrans-administration-portal) for more details.
-2. Select **Production** Environment from top-left environment switcher dropdown.
-    - refer to [Switching Environment](/en/midtrans-account/overview.md#switching-environment) for more details.
+2. To make sure your payment channels on Production account is active. 
+    - Go to menu **PAYMENT METHODS** from the dashboard page
+    - *Active Payment Methods* page is displayed.
```

Comment:
Is all merchants confirmed to have access to this page? AFAIK based on pilar issues, some manually onboarded merchant etc was unable to access this page. 
- maybe need confirmation to CMP?
- or maybe add notes to skip this step if they are unable to see the page?
- or maybe ask them to check snap payment method list?